src/Components/events/EditEvents.js

```diff
@@ -0,0 +1,90 @@
+import { useState, useEffect } from "react";
+import { Navigate, useParams } from "react-router-dom";
+
+import DeleteIcon from "@mui/icons-material/Delete";
+import DoneIcon from "@mui/icons-material/Done";
+
+import EditEventCard from "./EditEventCard";
+import AvatarUser from "../avatarUser/AvatarUser";
+import {
+  getEventsFromLocalStorage,
+  updateEvent,
+  getEventData,
+} from "../../utils";
+import "./styleEditEvents.scss";
+
+const EditEvent = () => {
+  const eventId = useParams().id;
+  const [event, setEvent] = useState(null);
+  const [itemToDraw, setItemToDraw] = useState(null);
+  const [backToDashboard, setBackToDashboard] = useState(false);
+
+  useEffect(() => {
+    async function getEvent(eventId) {
+      try {
+        const eventData = await getEventData(eventId);
+        const eventEdit = eventData.event;
+        setEvent(eventEdit);
+      } catch (error) {
+        console.log(error);
+      }
+    }
+
+    getEvent(eventId);
+  }, []);
+
+  useEffect(() => {
+    if (!event) {
+      return;
+    }
+    setItemToDraw(
+      <EditEventCard event={event} onChangeDataEvent={onChangeDataEvent} />
+    );
+  }, [event]);
+
+  const onChangeDataEvent = (eventEdit) => {
+    setEvent(eventEdit);
```

Comment:
since we don't have any logic to apply before calling `setEvent`, then we can use `setEvent` directly and pass it to the `EditEventCard`, you can remove `onChangeDataEvent` too